### ISSUE TITLE
ProgressIndicator is invisible in RecyclerView

### ISSUE THREAD

**Issue by eygraber** [material-components/material-components-android#1662]
**Description:** If I have a ProgressIndicator in a RecyclerView, and the ViewHolder's View gets detached from the window, when it gets reattached, the ProgressIndicator is invisible.

**Expected behavior:** It remains visible

**Android API version:** 29

**Material Library version:** 1.3.0-alpha02

**Device:** Pixel 2

**Comment by ahetawal-p** [comment]
This seems similar to my issue: https://github.com/material-components/material-components-android/issues/1681

**Comment by jrocharodrigues** [comment]
Did you find any workaround? I have the same problem...

**Comment by jrocharodrigues** [comment]
In version 1.3.0-alpha01 works fine

**Comment by hiberfake** [comment]
I can confirme that it works fine in version **1.3.0-alpha01**.

To solve this issue in version **1.3.0-alpha02**, I've overridden the following method in my `RecyclerView.Adapter`:
```kotlin
override fun onViewAttachedToWindow(holder: RecyclerView.ViewHolder) {
    holder.itemView.progressIndicator.visibility = View.VISIBLE
}
```

**Comment by ahetawal-p** [comment]
Looking into this more, this issue also happens if you have 2 recycler views inside a Tablayout. 
When you switch between the tabs, the progress bar disappears after the switch.

**Comment by ahetawal-p** [comment]
Bump. This bug is almost making the ProgressIndicator unusable in a recyclerview..

**Comment by hiberfake** [comment]
The issue still exists in version **1.3.0-alpha04**.

**Comment by pekingme** [comment]
Obsolete, as `ProgressIndicator` has been divided into `LinearProgressIndicator` and `CircularProgressIndicator`. Similar bugs are reported and fixed in #1914 and #1902.
